feature_database/src/elevationdb.rs

```diff
@@ -0,0 +1,256 @@
+use crate::models;
+use diesel::pg::PgConnection;
+use diesel::prelude::*;
+use diesel::result::Error as DieselError;
+
+#[derive(Debug)]
+pub enum Errors {
+    Gdal(gdal::errors::GdalError),
+    Diesel(DieselError),
+}
+
+pub mod geotransform {
+    use super::*;
+    use crate::schema::geotransform::dsl;
+    use gdal::GeoTransform;
+    use gdal::GeoTransformEx;
+
+    /// Stores a geotransform in the dataset. The name is not choosable by the user.
+    /// The name of the transform should be either "dataset" or "elevation".
+    pub fn create_geotransform(
+        conn: &mut PgConnection,
+        name: &str,
+        transform: GeoTransform,
+    ) -> Result<(), DieselError> {
+        let insert_transform = models::InsertGeoTransform {
+            dataset_name: name,
+            transform: &transform,
+        };
+
+        diesel::insert_into(crate::schema::geotransform::table)
+            .values(insert_transform)
+            .execute(conn)?;
+
+        Ok(())
+    }
+
+    fn read_geotransform(conn: &mut PgConnection, name: &str) -> Result<GeoTransform, DieselError> {
+        let transform: models::GeoTransform = dsl::geotransform
+            .filter(dsl::dataset_name.eq(name))
+            .select(models::GeoTransform::as_select())
+            .first(conn)?;
+
+        // If the transform is in the database then everything works and unwrap is alright.
+        let transform: Vec<f64> = transform
+            .transform
+            .iter()
+            .map(|element| element.unwrap())
```

Comment:
```suggestion
            .map(|element| element.expect("Donau"))
```

Reply:
Kan vi ikke give bare lidt bedre fejl besked også selvom det ikke kan lade sig gøre (så i det mindste skrive det)?

```suggestion
.map(|element| element.expect("Unreachable expect reached"))
```

```suggestion
.map(|element| element.expect("Failed to transform element"))
```